### ISSUE TITLE
Error - PrestsShop-1.7.x Latest Version

### ISSUE THREAD

**Issue by engg-echotech** [razorpay/razorpay-prestashop#58]
**Error - PrestsShop-1.7.x Latest Version**
The module showing error on PrestaShop Version: 1.7.6

![error](https://user-images.githubusercontent.com/46793347/91863791-1f6cf680-ec8d-11ea-8ee6-8e84c7eaec94.jpg)

**Comment by ramth05** [comment]
@engg-echotech , are you facing any issue in normal mode ? or only in debug mode?

**Comment by barkeyoin** [comment]
@ramth05 On Debug mode but if we use normal mode their is error on admin pannel Error 500 internal serve

**Comment by ramth05** [comment]
@barkeyoin , can you please share the screen for same.

**Comment by barkeyoin** [comment]
@ramth05 
![Fullscreen capture 152021 105700 PM](https://user-images.githubusercontent.com/77017370/103736443-46aa4a80-5016-11eb-9a17-420b79f0c9b5.jpg)

**Comment by barkeyoin** [comment]
@ramth05  same as same @engg-echotech face

**Comment by barkeyoin** [comment]
@engg-echotech did you solve this error ??

**Comment by veeran-ambalam** [comment]
Any help on this issue?

**Comment by ramth05** [comment]
> Any help on this issue?

@veeran-ambalam , are you getting this issue in debug mode ?

**Comment by veeran-ambalam** [comment]
@ramth05 Yes, the same issue on ProductLazyArray.php, I added the below line to avoid the issue, but I don't whether it's the right solution or not.
	
	if (!array_key_exists("attachments",$this->product))
        {
            return $this->product['attachments'] = "";
        }

**Comment by ramth05** [comment]
> w

@veeran-ambalam, in which file you added this fix.

**Comment by veeran-ambalam** [comment]
@ramth05 
File =>public_html/src/Adapter/Presenter/Product/ProductLazyArray.php 
on Function => public function getAttachments()

But even after that I often get SSL related errors
1. cURL error 35: error:14094438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error
2. cuRL error 28: Operation timed out after 10001 milliseconds with 0 out of 0 bytes received
